modules/core/test/v2/unit/coins/trx.ts

```diff
@@ -84,6 +84,24 @@ describe('TRON:', function() {
     signedTxJson.signature[0].should.equal('0a9944316924ec7fba4895f1ea1e7cc95f9e2b828ae268a48a8dbeddef40c6f5e127170a95aed9f3f5425b13058d0cb6ef1f5c2213190e482e87043691f22e6800');
     });
 
+  it('should sign with an Xprv a half signed tx', () => {
+    const p = {
+      prv: "xprv9s21ZrQH143K2sg2Cukk5XqLQdrYnMCDah3y1FFVy6Hz9bQfqMSfmUiHPVHKhcUyft3N1emE5FudJVxgFm5N12MAg5o7DTPsDATTkwNgr73",
+      txPrebuild: {
+        txHex: signTxOptions.txPrebuild.txHex
+      }
+    };
+    const tx = basecoin.signTransaction(p);
+    const unsignedTxJson = JSON.parse(signTxOptions.txPrebuild.txHex);
+    const signedTxJson = JSON.parse(tx.halfSigned.txHex);
+
+    signedTxJson.txID.should.equal(unsignedTxJson.txID);
+    signedTxJson.raw_data_hex.should.equal(unsignedTxJson.raw_data_hex);
+    JSON.stringify(signedTxJson.raw_data).should.eql(JSON.stringify(unsignedTxJson.raw_data));
```

Comment:
hmm i don't think we need this check if we have the check above... should be the same and not mutate the tx

Reply:
I think it does make sense. We want to make sure that the structure of the tx doesn't change, but only the signature if it is a different format.
May not be the most important part of the tests for us that we know how it works, but I don't think it is extra.